src/Stripe.net/Infrastructure/Public/StripeTypeRegistry.cs

```diff
@@ -21,6 +21,7 @@ public static class StripeTypeRegistry
             { "balance", typeof(Balance) },
             { "balance_transaction", typeof(BalanceTransaction) },
             { "bank_account", typeof(BankAccount) },
+            { "billing_portal.session", typeof(BillingPortal.Session) },
```

Comment:
not quite sure how this bit works, but it seems to match the issuing ones, so lgtm